src/types/adaptivecard.ts

```diff
@@ -15,4 +15,5 @@ export interface IAdaptiveCardProps {
     getAdaptiveCard: Function;
   };
   queryStatus?: any;
+  styles?: object;
```

Comment:
This should not be necessary since messages/theme/style are passed down automatically